_ml-commons-plugin/model-serving-framework.md

```diff
@@ -0,0 +1,250 @@
+---
+layout: default
+title: Model serving framework 
+has_children: false
+nav_order: 110
+---
+
+# Model serving framework
+
+ML commons supports the ability to serve custom models and use those models to make inferences. For those who wish to use train their models using a deep-learning framework, such as Pytorch, they can upload such models to OpenSearch in order to take advantage of OpenSearch resources.
```

Comment:
```suggestion
ML commons supports the ability to serve custom models and use those models to make inferences. For those who wish to train their models using a deep-learning framework, such as Pytorch, they can upload such models to OpenSearch in order to take advantage of OpenSearch resources.
```

Reply:
Make predictions?

Reply:
This sentences seems not accurate "For those who wish to use train their models using a deep-learning framework, such as Pytorch, they can upload such models to OpenSearch in order to take advantage of OpenSearch resources."

How about change to 
"For those who wish to run their trained deep-learning model (only support PyTorch now) inside OpenSearch cluster, they can upload and run model with ml-commons REST API."

Reply:
Inferences, which would be a step before prediction.